client/apps/edit/components/section_tool/index.coffee

```diff
@@ -19,6 +20,15 @@ module.exports = React.createClass
   isAboveTextSection: ->
     @props.sections.models[@props.index + 1]?.get('type') is 'text'
 
+  getJumpLinks: ->
+    links = []
+    @props.sections.each (section) ->
```

Comment:
I think it'd be more appropriate to add this as a method on the Sections collection actually.